src/components/application_manager/src/display_capabilities_builder.cc

```diff
@@ -152,7 +154,27 @@ bool DisplayCapabilitiesBuilder::IsWaitingForWindowCapabilities(
 void DisplayCapabilitiesBuilder::ResetDisplayCapabilities() {
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock lock(display_capabilities_lock_);
-  display_capabilities_.reset();
+  for (auto& window_id : window_ids_to_resume_) {
+    if (kDefaultWindowID != window_id) {
+      window_ids_to_resume_.erase(window_id);
+    }
+  }
+
+  if (display_capabilities_) {
+    auto cur_window_caps =
+        (*display_capabilities_)[0][strings::window_capabilities].asArray();
+    if (cur_window_caps) {
+      std::remove_if(cur_window_caps->begin(),
```

Comment:
Are we modifing cur_window_caps as part of *display_capabilities_ ? 
it is not clear, Could you please make `auto& cur_window_caps` or `auto* cur_window_caps` to make it explicit

Reply:
@LuxoftAKutsan yes. Fixed in e963dbd